src/prng.rs

```diff
@@ -219,19 +219,20 @@ mod tests {
             .collect()
     }
 
+    #[ignore]
```

Comment:
Reviewer note: We can either update this test or replace it with one for SHA-3. What would folks prefer?

Reply:
I think taking a TODO to swap in the SHA-3 test once we implement its PRG would be best. I could also re-run the brute-force search in short order if need be, I have the script for it lying around still.

Reply:
Filed #465 to track this.